### DISCUSSION TITLE
ch8/ch8-01

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#167] [General]
# ch8/ch8-01

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch8/ch8-01.html

**Comment by zhangbest5** [comment]
长见识了，还有\r这么个覆盖的用法

**Comment by logn-xu** [comment]
%c这个没看懂  看解释是打印 %c：打印对应的的unicode码值

**Comment by GingerNan** [comment]
最后一句话说spinning和菲波那契的计算的函数同时执行。是因为spinner函数是goroutine所以才同时执行的。是嘛？

**Reply by hpf0532** [reply]
没错

**Comment by lazybearlee** [comment]
\r 使光标移到行首
\n 换行，使光标下移一格
%c 该值对应的unicode码值（用于整型）

**Comment by Etherealss** [comment]
`\r` 是一个转义字符，表示回车（Carriage Return）。它的作用是将光标移动到当前行的开头，而不换行。这意味着在输出新的字符时，会覆盖当前行的内容。
